USAGE.rst

```diff
@@ -14,6 +14,7 @@ Preparation
 
 First, decide where to store input files, a good place might be ``$HOME`` (``~``) because it is usually backed up.
 Second, decide where to store your output of simulations which needs to be placed on a high-bandwidth, large-storage file system which we will refer to as ``$SCRATCH``.
+(For a first test you can also use your home directory as simulation directory. Just make sure the available disk space is large enough.)
```

Comment:
I am not a big fan of making a manual verbose when it is clear. Nevertheless, if you want to be overexplicit then let us:

- add a newline to make a new paragraph
- No "()"
- Add keywords: For a first test you can also use your home directory `$HOME` as simulation directory `$SCATCH`. ~~Just make sure the available disk space is large enough.~~ -> Remove since it does not help if you are not saying *how large*.